### ISSUE TITLE
Foundry V11 compatibility

### ISSUE THREAD

**Issue by iotech-fvtt** [p4535992/foundryvtt-inventory-plus#35]
Please consider re-working for V11 compatibility

**Comment by Khorius** [comment]
Would definitely love to see this updated. There is no better module for handling bags of holding.

**Comment by baal000** [comment]
I just tested it superficially and it does work (at least at a glance) if you increase the maximum Version in the module.json to 11

**Comment by Khorius** [comment]
it does "work", it just has some issues that cause it to break. If you make a new custom category, you won't be able to delete it or anything.

**Comment by DMSei** [comment]
Is this a dead Module? No one has accepted the ticket or responded for ages from a project management position. I too would like to see this updated.

**Comment by zero127** [comment]
As of the last foundry update this can no longer edit some categories randomly. Uncaught promise in label error in console. Really wish somebody would update this, its crucial to how we play the game T_T

**Comment by Maradeurfv** [comment]
+1

**Comment by p4535992** [comment]
for anyone innterested there is a open commission on the foundryvtt league server

**Comment by KonteiKeisei** [comment]
Yeah, i have been following that and it was hijacked away by CCSS mod, which does not actually do what this mod does.
Considering KGAR rewrote Tidy5e and has it in alpha, I +1 this request.

**Comment by DreamyNiri** [comment]
yea this defs needs an update, the only module which does this and quite a popular one too
